packages/contracts/test/manager/facets/ManagerFacet.t.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../DiamondTestSetup.sol";
+import "../../../src/dollar/interfaces/ICurveFactory.sol";
+import "../../../src/dollar/interfaces/IMetaPool.sol";
+import "../../../src/dollar/mocks/MockDollarToken.sol";
+import "../../../src/dollar/mocks/MockTWAPOracleDollar3pool.sol";
+
+import {GOVERNANCE_TOKEN_MINTER_ROLE, GOVERNANCE_TOKEN_BURNER_ROLE} from "../../../src/manager/libraries/LibAppStorage.sol";
+
+contract TestManagerFacet is DiamondSetup {
```

Comment:
IMO we need other tests 
- one where we add a facet that reads from appstorage
- one that add a field to appstorage struct 
- one we a new facet that need an init function call

Reply:
> * one where we add a facet that reads from appstorage

ManagerFacet is already doing it